ts/features/pn/components/__test__/MessageDetailsContent.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render } from "@testing-library/react-native";
+import { MessageDetailsContent } from "../MessageDetailsContent";
+
+describe("MessageDetailsContent component", () => {
+  it("should match the snapshot when abstract is defined", () => {
```

Comment:
Should we add a snapshot test with undefined abstract?

Reply:
Applied https://github.com/pagopa/io-app/pull/5431/commits/3efbf44c589ca4c843609d0a82613db12aec9439